superset/connectors/druid/models.py

```diff
@@ -798,6 +798,28 @@ def values_for_column(self,
     def get_query_str(self, query_obj, phase=1, client=None):
         return self.run_query(client=client, phase=phase, **query_obj)
 
+    def _add_filter_from_pre_query_data(self, df, dimensions, filter):
+        ret = filter
```

Comment:
`filter` is a reserved word which can be confusing, go for `filter_` or `filters`

Reply:
ah... keep forgetting filter :(